docs/authmodel.md

```diff
@@ -0,0 +1,58 @@
+# Examples of organization structures modeled with Udaru
+
+Access to resources can be modeled in Udaru through the elements that it provides: Organizations, Teams, Users. Policies can be attached to any of these elements.
+See more details in the [Authorization Introduction][]
+
+The usual modeling case is to build independent organizations, each organization having teams (or nested teams) attached to them and users attached to teams. Policies are attached to Organizations, Teams or Users.
+
+In the examples below in the **Cross organization access management** section it is described a particular case in which we need users that have access rights over several organizations, this corner case not matching exactly the Udary modeling architecture.
```

Comment:
`this corner case is not a perfect match for the Udaru architecture, but we can achieve it using the shallow impersonation feature` ?